### DISCUSSION TITLE
Sector Nebula

### DISCUSSION THREAD

**Discussion by dendyua** [Yaro2709/New-Star#51] [Show and tell]
When the fleet is sent to the Nebula Sector, when the event time is reached, nothing happens. The return time of the fleet becomes negative!

**Comment by Yaro2709** [comment]
Version php?

**Reply by dendyua** [reply]
PHP 5.3.13
*/
Apache 2.2.22
MySQL 5.5.25 (5.0.8)
/*

**Reply by Yaro2709** [reply]
When you installed the game, you didn't see that it was written no lower than php7.0?

**Reply by Yaro2709** [reply]
https://i2.paste.pics/b68d4411b41e75ccdae2c4ea3d0b8649.png

**Reply by dendyua** [reply]
> When you installed the game, you didn't see that it was written no lower than php7.0?

Yes, I saw, I decided to take a chance.My work computer uses Windows XP so there is no way to install newer versions.
